tests/unit/lms/services/canvas_test.py

```diff
@@ -225,6 +235,9 @@ def test_find_matching_file_in_course_with_multiple_file_ids(
             # The third file_id *will* be found in the course.
             matching_file,
         ]
+        # The with GUID fallback doesn't returning anything else
+        file_service.get_with_guid.return_value = None
```

Comment:
I could probably re-write this test to parameterize the return values of both `file_service` methods and remove the new test.

I rather do that in a follow up PR.